index.js

```diff
@@ -240,26 +247,26 @@ class Link {
 
     this._monitorItv = setInterval(this.monitor.bind(this), this.conf.monitorTimeout)
 
-    _.each(['lookup'], fld => {
-      const cfld = _.upperFirst(fld)
+    for (const fld of ['lookup']) {
+      const cfld = fld.charAt(0).toUpperCase() + fld.substring(1)
 
       const opts = {
         max: this.conf[`lruMaxSize${cfld}`],
         maxAge: this.conf[`lruMaxAge${cfld}`]
       }
 
       this.cache[fld] = new LRU(opts)
-    })
+    }
 
     return this
   }
 
   stop () {
     clearInterval(this._monitorItv)
 
-    _.each(this.cache, c => {
-      c.clear()
-    })
+    for (const key in this.cache) {
+      this.cache[key].clear()
```

Comment:
```suggestion
    for (const entry in Object.values(this.cache)) {
      entry.clear()
```

As a standard we should avoid using `for ... in` on objects because of inherited prototype properties